xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererDRMPRIMEGLES.cpp

```diff
@@ -0,0 +1,220 @@
+/*
+ *      Copyright (C) 2007-2015 Team XBMC
+ *      http://xbmc.org
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with XBMC; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "RendererDRMPRIMEGLES.h"
+
+#include "cores/VideoPlayer/DVDCodecs/DVDCodecUtils.h"
+#include "cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecDRMPRIME.h"
+#include "cores/VideoPlayer/VideoRenderers/RenderFactory.h"
+#include "ServiceBroker.h"
+#include "utils/log.h"
+
+#define BUFFER_OFFSET(i) ((char *)NULL + (i))
+
+static CWinSystemGbmGLESContext *m_pWinSystem{nullptr};
```

Comment:
Don't know if it's been done like that in other places, but prefix `m_` for a global variable seems very doubtful to me

Reply:
are you just worried about the naming convention?

Reply:
Basically, yes. `m_` indicates a member variable, which this clearly isn't. If we do Hungarian notation, we should at least apply it consequently.

btw I also don't like the global variable, but that's a bigger problem and not particular to this PR I think

Reply:
please check it now :)

Reply:
Heh it's basically the same thing in a different package ;) But the `m_` is alright now :+1: